synctl

```diff
@@ -668,8 +668,12 @@ class SyntheticConfiguration(Base):
             self.syn_test_config["locations"] = locations
 
     def set_timeout(self, timeout: int) -> None:
-        """timeout"""
-        self.syn_test_config["configuration"]["timeout"] = timeout
+        """timeout <number>(ms|s|m)"""
+        if str(timeout).endswith(("m", "s", "ms")):
+            self.syn_test_config["configuration"]["timeout"] = timeout
```

Comment:
One issue of the checking, for example `5mm`, `5ss`, and `5msms`. 
I test the command, API always reports an error if the timeout is not valid value. I think we can send the value to API and we don't check it from our side. I will update the card.